src/main/java/net/sf/jabref/Globals.java

```diff
@@ -61,8 +61,7 @@ public static void startBackgroundTasks() {
         Globals.streamEavesdropper = StreamEavesdropper.eavesdropOnSystem();
 
         Globals.fileUpdateMonitor = new FileUpdateMonitor();
-        JabRefExecutorService.INSTANCE.executeWithLowPriorityInOwnThread(Globals.fileUpdateMonitor,
-                "FileUpdateMonitor");
+        JabRefExecutorService.INSTANCE.executeInterruptableTask(Globals.fileUpdateMonitor);
```

Comment:
This shouldn't be a problem. Maybe it makes sense for the specific threads, which are very prominent. Using an executor makes lifecycle management a little nicer at least.

I have to look into the remote thread as well, I guess it makes sense for that one, too.